pepys_import/core/store/data_store.py

```diff
@@ -1644,3 +1618,12 @@ def get_logs_by_change_id(self, change_id):
             .filter(self.db_classes.Log.change_id == change_id)
             .all()
         )
+
+    def get_measurement_tables(self):
+        return list(self.meta_classes[TableTypes.MEASUREMENT])
+
```

Comment:
Hello - you're going to be cross with me here, I'm not trying to annoy you 🤣 

I've just realised we can remove even more duplication.

if the methods return a `TableTypes` enum value, then in `get_status()` we do:
`tables = list(self.meta_classes[tableType])`

Hmm, actually it can get even simpler. We no longer need those 3 methods.  We just pass `TableTypes.MEASUREMENT` to `get_status()`.

Reply:
I was trying to find a middle way between your suggestion and minimum change of code (without breaking anything)😄 Yes, we can directly pass table type. I'm changing `report()` now. After that, I'll change `get_status()` again.